test/cli/seed.spec.js

```diff
@@ -20,6 +20,16 @@ describe('seed:run', () => {
     );
   });
 
+  it('find recursively files', () => {
+    return execCommand(
+      `node ${KNEX} seed:run --knexfile=test/jake-util/seeds-knexfile-directories.js`,
+      {
+        expectedOutput: 'Ran 3 seed files',
```

Comment:
would be helpful to also assert expected seed names

Reply:
Added